frontend/src/pages/projects/screens/detail/DetailsSection.tsx

```diff
@@ -75,7 +75,7 @@ const DetailsSection: React.FC<DetailsSectionProps> = ({
               {title}
             </Title>
           </FlexItem>
-          {actions && <FlexItem>{actions}</FlexItem>}
+          {actions && <FlexItem style={{ display: 'flex' }}>{actions}</FlexItem>}
```

Comment:
Is this not just `Flex`?

Reply:
```suggestion
          {actions && (
            <Flex direction={{ default: 'row' }}>
              {actions.map((action, index) => (
                <FlexItem key={index}>{action}</FlexItem>
              ))}
            </Flex>
          )}
```
Something like this seems to work. Providing custom styles is a cop-out and likely removes responsibility to do the right thing. Please avoid using direct styles unless it's a PF fault.

Reply:
Thanks, it works, I added a `spaceItems={{ default: 'spaceItemsSm' }}` to make the space between the button and kebab icon look better (or closer to what it looked like before).